development/build/transforms/remove-fenced-code.js

```diff
@@ -337,7 +337,16 @@ function removeFencedCode(filePath, typeOfCurrentBuild, fileContent) {
       // Throws an error if the command parameters are invalid
       CommandValidators[command](parameters, filePath);
 
-      if (parameters.includes(typeOfCurrentBuild)) {
+      const validBuildTypes = [
+        typeOfCurrentBuild,
+        ...(BuildTypeInheritance[typeOfCurrentBuild] || []),
```

Comment:
This build inheritance idea is great! Really simplified this PR.

Not critical, but it would be nice to have a targeted unit test to ensure it's working correctly. I see that it's covered already in that large test fixture (which is fantastic and very much appreciated), but to somebody unfamiliar with this code, this behavior might not be obvious.

Having a targeted test with a minimal input _just_ to cover this build type inheritance idea would do a better job of ensuring this is understood and continues to work in the future.

Reply:
Great idea, done.